### ISSUE TITLE
Android app crashes when attaching image

### ISSUE THREAD

**Issue by FinlayDaG33k** [devRant/devRant#471]
When trying to attach an image, the Android app crashes quite often on an LG Nexus 5X with Android 8.0

This could tie-in with #443 ?

**Comment by Skayo** [comment]
Hard to reproduce :/
Does this happen every time? Can you try other images or other sources?

**Comment by FinlayDaG33k** [comment]
Happens *nearly* everytime (unless I'm too slow, then it still crashes).

Tried loads of images over the course of a few months (local storage on device),  always kept happening if I waited too long.

**Comment by Skayo** [comment]
Could be an issue of Appcelerator... 
Does posting images work in other apps?

**Comment by FinlayDaG33k** [comment]
Yes, it works just fine in other apps (Instagram, Whatsapp, Telegram, Signal and Protonmail just to name a few)

**Comment by FinlayDaG33k** [comment]
If you could provide me with a way to do so, then I probably could.

**Comment by Skayo** [comment]
@FinlayDaG33k I think it's possible with ADB Logcat. Never used it tho...

**Comment by FinlayDaG33k** [comment]
I've tried it but couldn't get it to work :\

**Comment by Skayo** [comment]
Not sure if the "Give Feedback" feature helps there. Would need to ask dfox for that.

**Comment by cozyplanes** [comment]
@Skayo happens frequently here

As far as i know, give feedback sends a log to the google developer console

**Comment by Skayo** [comment]
If that's the case I'll mention him: @DFoxinator
